components/gitpod-db/src/blocked-repository-db.ts

```diff
@@ -9,5 +9,15 @@ import { BlockedRepository } from "@gitpod/gitpod-protocol/src/blocked-repositor
 export const BlockedRepositoryDB = Symbol("BlockedRepositoryDB");
 
 export interface BlockedRepositoryDB {
+    findAllBlockedRepositories(
+        offset: number,
+        limit: number,
+        orderBy: keyof BlockedRepository,
```

Comment:
Here it might make sense to limit to the fields that actually are indexex, for instance by using `orderBy: keyof Pick<BlockedRepository, "id | "blockUser">` or similar.

Reply:
We have no indexes on the table besides the PK. As mentioned yesterday https://github.com/gitpod-io/gitpod/pull/11036#discussion_r910989383, mysql can't make use of indexes on `urlRegexp` when testing with `REGEXP`.

Do you think we should add indexes to support this admin search?